utils/data-migration/gef/map-facilities.js

```diff
@@ -68,7 +68,7 @@ const mapCoverEndDate = (coverStartDate, exposurePeriod) => {
     months: exposurePeriod,
   });
 
-  return getUnixTime(date);
+  return new Date(getUnixTime(date));
```

Comment:
We are not storing date as EPOCH?

Reply:
unfortunately not for cover end date.

Reply:
Ah okay, can we store it as EPOCH or is it going to be a big job to do so and out of scope for this PR?